packages/block-editor/src/hooks/style.js

```diff
@@ -202,6 +243,45 @@ export const withBlockControls = createHigherOrderComponent(
 	'withToolbarControls'
 );
 
+/**
+ * Override the default block element to include duotone styles.
+ *
+ * @param  {Function} BlockListBlock Original component
+ * @return {Function}                Wrapped component
+ */
+const withElementsStyles = createHigherOrderComponent(
+	( BlockListBlock ) => ( props ) => {
+		const elements = props.attributes.style?.elements;
+		if ( ! elements ) {
+			return <BlockListBlock { ...props } />;
+		}
+		const blockElementsContainerIdentifier = `wp-block-elements-container-${ useInstanceId(
```

Comment:
I think we should use [the same as in the server](https://github.com/WordPress/gutenberg/pull/31524/files#diff-b85df651c33208862897162a87698bad76f15535f6432b97b022a72e99230edeR29): `'wp-elements-ID`